src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/TestsConnectionFactory.cs

```diff
@@ -65,8 +93,27 @@ DbConnection IConnectionFactory.CreateConnection(string connection)
         }
     }
 
-    internal class SqlServerTestsConnectionFactory : IConnectionFactory
+    internal class SqlServerTestsConnectionFactory : IConnectionFactory, ITestsDbConnectionFactory
     {
+        public string ConnectionString => "Data Source=.;Initial Catalog=SFA.DAS.ApprenticeCommitments.AcceptanceTests;Integrated Security=True;";
+
+        public void EnsureCreated(ApprenticeCommitmentsDbContext dbContext)
+        {
+            if (!dbContext.Database.CanConnect())
+            {
+                throw new Exception(
+@"SQL Server Acceptance Test database not found.
+
+To ensure that we are testing against the real database schema used in production, do not rely on EFCore to create the database.
+
+Instead, manually deploy the database using the `SFA.DAS.ApprenticeCommitments.Database` project, targetting `SFA.DAS.ApprenticeCommitments.AcceptanceTests`");
+            }
+        }
+
+        public void EnsureDeleted(ApprenticeCommitmentsDbContext dbContext)
+        {
+        }
```

Comment:
Perhaps this EnsureDeleted should actually clear up the database by Truncating all the tables. It's quicker that DELETE and I don't think checks contstraints and we can't roll back. But it can be done another time

Reply:
Yes, I did think that too but never implemented it.  I'll leave it for now.